src/rtpghi~.c

```diff
@@ -110,8 +91,9 @@ void *rtpghi_tilde_new(t_symbol *s, int argc, t_atom *argv)
     
     post("%p", (x->sta_pd));
     int init_s = 0;
-    
-    init_s = phaseret_rtpghi_init_s(gamma, W, a , M, tol, do_causal, &(x->sta_pd));
+	ltfat_int w = 1;
+	
+    init_s = phaseret_rtpghi_init_s(gamma, w, a , M, tol, do_causal, &(x->sta_pd));
```

Comment:
also, `phaseret_rtpghi_init` allocates a new phaseret_state, so you have any previously allocated state.

Reply:
please resolve these issues with (pushed) commits, and not by simply marking them as "resolved" (alternatively, add a comment why you think it ought not be resolved)